packages/server/config/teamspeak.sample.js

```diff
@@ -1,12 +1,22 @@
 module.exports = {
```

Comment:
- This configuration section is not documented!
- All these are breaking changes!